### PR TITLE
Feature ci keoken

### PR DESCRIPTION
**Pull request by fpelliccioni**

### CODE REVIEW

File: src/bitprim/keoken/interpreter.cpp [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+    if ( ! fast_chain_.get_output(out_output, out_height, out_median_time_past, out_coinbase, 
+                                  owner_input.previous_output(), libbitcoin::max_size_t, true)) {
+        return payment_address{};
+    }
+
+    return out_output.address();
+}
+
+error_code_t interpreter::process_create_asset_version_0(size_t block_height, transaction const& tx, reader& source) {
+    auto msg = message::create_asset::factory_from_data(source);
```
Comment (mdallago) [inline src/bitprim/keoken/interpreter.cpp:101]: missing msg construction validation

File: src/bitprim/keoken/interpreter.cpp [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+
+    if (it == tx.outputs().end()) {
+        return {std::move(source), payment_address{}};        
+    }
+
+    return {std::move(source), it->address()};
+}
+
+error_code_t interpreter::process_send_tokens_version_0(size_t block_height, transaction const& tx, reader& source) {
+    auto msg = message::send_tokens::factory_from_data(source);
```
Comment (mdallago) [inline src/bitprim/keoken/interpreter.cpp:135]: missing msg construction validation

File: src/bitprim/keoken/interpreter.cpp [hunk 3]
```diff
@@ -0,0 +1,473 @@
...
+
+    state_.create_balance_entry(msg.asset_id(), msg.amount(), source_addr, target_addr, block_height, tx.hash());
+    return error::success;
+}
+
+} // namespace keoken
+} // namespace bitprim
+
+
+
```
Comment (mdallago) [inline src/bitprim/keoken/interpreter.cpp]: remove comments

File: src/bitprim/keoken/interpreter.cpp (hunk 1 above)
Comment (fpelliccioni) [reply src/bitprim/keoken/interpreter.cpp:101]: Fixed in `bitprim-core` 
https://github.com/bitprim/bitprim-core/pull/37

File: src/bitprim/keoken/interpreter.cpp (hunk 2 above)
Comment (fpelliccioni) [reply src/bitprim/keoken/interpreter.cpp:135]: fixed in `bitprim-core`
https://github.com/bitprim/bitprim-core/pull/37

File: src/bitprim/keoken/interpreter.cpp (hunk 3 above)
Comment (fpelliccioni) [reply src/bitprim/keoken/interpreter.cpp]: fixed

**Review by mdallago — APPROVED** [review]

**Pull request merged**